compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1512,6 +1537,31 @@ OMR::ResolvedMethodSymbol::supportsInduceOSR(TR_ByteCodeInfo &bci,
    return true;
    }
 
+bool
+OMR::ResolvedMethodSymbol::supportsInduceOSR(TR::Node* osrNode,
```

Comment:
why do we need this rather than isPotentialOSRPoint or isPotentialOSRPointWithSupport?